api/app/GraphQL/Queries/CountPoolCandidatesByPool.php

```diff
@@ -70,8 +70,8 @@ public function __invoke($_, array $args)
             }
 
             // wouldAcceptTemporary
-            if (array_key_exists('wouldAcceptTemporary', $filters)) {
-                User::scopeWouldAcceptTemporary($userQuery, $filters['wouldAcceptTemporary']);
+            if (array_key_exists('positionDuration', $filters)) {
```

Comment:
don't forget to update the comment

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/4956/commits/3500ed3b81d9c0c2d4d3ccce5d3251b1bc7f0afe
done it